substrate/frame/support/src/traits/tokens/fungibles/regular.rs

```diff
@@ -117,6 +117,9 @@ pub trait Inspect<AccountId>: Sized {
 
 	/// Returns `true` if an `asset` exists.
 	fn asset_exists(asset: Self::AssetId) -> bool;
+
+	/// Returns `Some(true)` if an `asset` exists and is sufficient.
```

Comment:
Should it return `None` if the asset does not exist? And `Some(false)` if it exists and is not sufficient?
A better doc would be useful here

Reply:
You are right. It's done.

The new doc states the following:
```rs
	/// Returns `Some(true)` if an `asset` exists and is sufficient. Returns `Some(false)`
	/// if an `asset` exists, but is not sufficient. Returns `None` if an `asset` does not exist.
```